packer_test/core_tests/local_eval_test.go

```diff
@@ -21,8 +21,8 @@ func (ts *PackerCoreTestSuite) TestEvalLocalsOrder() {
 }
 
 func (ts *PackerCoreTestSuite) TestLocalDuplicates() {
-	pluginDir, cleanup := ts.MakePluginDir()
-	defer cleanup()
+	pluginDir := ts.MakePluginDir()
+	defer pluginDir.Cleanup()
```

Comment:
My on nit here is that by making this a method on pluginDir as opposed to an anonymous returned function is that you can easily change the value of pluginDir.dirPath and remove some other directory.